### PR TITLE
modified lists of keys to set.

### PR DESCRIPTION
**Pull request by makarman**
## JIRA

* [Main JIRA ticket](https://ssu-jira.softserveinc.com/secure/RapidBoard.jspa?rapidView=738)

## Code reviewers

- [ ] @dmytrorevak
- [ ] @IvanAndrukh
- [ ] @JjuliaP
- [ ] @makarman
- [ ] @noobhunters
- [ ] @Silent777
- [ ] @Staryj812

### Second Level Review

- [ ] @ssOleg
- [ ] @lhalam

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: eventually/utils/validators.py [hunk 1]
```diff
@@ -98,11 +98,10 @@ def required_keys_validator(data, keys_required, strict=True):
 
     :return: `True` if data is valid and `False` if it is not valid.
```
Comment (skramar) [inline eventually/utils/validators.py:99]: keys_required param = list 
change it in docs to set

Comment (makarman) [reply eventually/utils/validators.py:99]: okey

File: eventually/utils/validators.py [hunk 2]
```diff
@@ -89,20 +89,18 @@ def required_keys_validator(data, keys_required, strict=True):
...
-    keys = list(data.keys())
-
+    keys = set(data.keys())
+    keys_required = set(keys_required)
     if strict:
-        keys.sort()
-        keys_required.sort()
         return keys == keys_required
 
     for key in keys_required:
```
Comment (lhalam) [inline eventually/utils/validators.py]: https://docs.python.org/2/library/sets.html#set-objects
use difference

File: eventually/utils/validators.py [hunk 3]
```diff
@@ -89,20 +89,18 @@ def required_keys_validator(data, keys_required, strict=True):
...
     :return: `True` if data is valid and `False` if it is not valid.
     """
-    keys = list(data.keys())
-
+    keys = set(data.keys())
+    keys_required = set(keys_required)
     if strict:
-        keys.sort()
-        keys_required.sort()
         return keys == keys_required
```
Comment (lhalam) [inline eventually/utils/validators.py]: use symmetric_difference

File: eventually/utils/validators.py (hunk 2 above)
Comment (makarman) [reply eventually/utils/validators.py]: done

File: eventually/utils/validators.py (hunk 3 above)
Comment (makarman) [reply eventually/utils/validators.py]: done

**Review by redmyt — APPROVED** [review]

**Pull request merged**